templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -106,30 +106,46 @@ const initNavigationSliding = () => {
   toggles.forEach(function (toggle) {
     toggle.addEventListener('click', function (e) {
       e.preventDefault();
+      closeSearch();
       const target = document.getElementById(toggle.getAttribute('aria-controls'));
       if (target) {
-        const isNested = target.parentNode.parentNode.classList.contains('p-navigation__dropdown');
-
+        const isNested = !target.closest('.p-navigation__dropdown');
```

Comment:
In this particular case, it's not really a JS hook, we are checking if specific dropdown is on top level or not, so it kind of relies on checking parent CSS class name.

Reply:
I'd want to avoid refactoring the whole script (that was not written for this feature) as part of this PR.

Because it's not about just this place here. The whole script would have to be significantly rewritten to remove the references of CSS class names or specific parent nodes.